Sample-WPF/MainWindow.xaml.cs

```diff
@@ -97,6 +97,10 @@ public MainWindow()
                     Title = "Face Verification",
                 },
             };
+
+            //Set the default endpoint when main windows is initiated
+            MainWindow mainWindow = Window.GetWindow(this) as MainWindow;
+            mainWindow._scenariosControl.SetSubscriptionPageEndpoint("https://westcentralus.api.cognitive.microsoft.com/face/v1.0");
```

Comment:
Sorry for not being clear before, but I think we should set `westus` as default endpoint region. The reason is that before GA of Face API, our cloud service only locates at `westus` and at that time there is no different endpoints or regions. So those users may not have the concept of endpoint or region. To make the sample user-friendly, I prefer to have the default value work for the old users and make the endpoint configurable for new users who are supposed to clearly know their subscription endpoints.